README.md

```diff
@@ -84,9 +98,12 @@ Normally you will only need to install Themis Core [from repositories](https://d
 This example project already includes prebuilt Themis JNI library for modern 64-bit Linux and macOS systems.
 
 If this does not work, you may need to build Themis from source code:
-1. Follow [these instructions](https://docs.cossacklabs.com/pages/documentation-themis/#java-wrapper-installation) to build Themis JNI library.
-2. Copy JavaThemis source code (`src/wrappers/themis/java`) into the `java-example/src` directory.
-3. Copy Themis JNI library (`build/libthemis_jni.*`) into the `java-example/lib` directory.
+1. Follow [these instructions](https://docs.cossacklabs.com/themis/languages/java/installation-desktop/) to build Themis JNI library.
+   You will also need to build the JAR.
+2. Copy Themis JNI library (`build/libthemis_jni.*`) into the `java-example/lib` directory.
```

Comment:
Just read the Themis main Makefile. If I'm correct, JNI lib is a part of Themis `.deb`, `.rpm` packages. So I guess you're right and I should have removed these `.so`, `.dylib` libs instead.

Reply:
> If I'm correct, JNI lib is a part of Themis .deb, .rpm packages.

You absolutely are. There is (now) a package with JNI libs for Linux. (There wasn't one back when these examples were created.) Homebrew for macOS also has an option to install the JNI library in addition to the Core libs.

Reply:
So, should I now remove both Linux & macOS libs, only leave the JAR and update README to mention installing not only Themis Core, but also JNI? And then, make this PR non-draft

Reply:
> should I now remove both Linux & macOS libs, only leave the JAR and update README to mention installing not only Themis Core, but also JNI

I think so, yeah. It should be enough to link to the installation guide and instruct to install the JNI library (it pulls in Core automatically, as a dependency). For now, as noted in the guide, the users have to build JAR themselves, but here in the example we already did it. Later it will be necessary to only install the JNI library.